packages/model/addon/-private/model.js

```diff
@@ -2157,7 +2164,7 @@ Model.reopenClass({
       callback.call(binding, name, meta);
     });
   },
-
+  // TODO refactor for native classes/octane??
```

Comment:
👍

Reply:
The *.extend is the correct syntax for native classes so no refactoring required